cargo/jam/commands/update_builder.go

```diff
@@ -49,7 +49,9 @@ func (ub UpdateBuilder) Execute(args []string) error {
 		for j, order := range builder.Order {
 			for k, group := range order.Group {
 				if group.ID == image.Path {
-					builder.Order[j].Group[k].Version = image.Version
+					if builder.Order[j].Group[k].Version != "" {
+						builder.Order[j].Group[k].Version = image.Version
+					}
```

Comment:
I don't understand this change. Does this mean that if I had a version declared in my builder, which we do in almost every case, we wouldn't update it?

Reply:
The version field in the order grouping is optional according to [these docs](https://buildpacks.io/docs/reference/config/builder-config/#order-_list-required_). This change makes it so that if you do not have a version specified, when you run `update-builder`, a version does not get added. The version only gets updated now when the order group entry already had a version.

The original implementation *always* adds a version field with the most current buildpack version, even when the original builder.toml didn't have a version. This isn't a huge issue, but since the field is optional, our script shouldn't be adding a new field.